plaso/parsers/sqlite_plugins/ios_powerlog.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+"""SQLite parser plugin for iOS powerlog database files. It can be obtained from
+an iOS sysdiagnose dump at <sysdiagnose_DATE_OS_DEVICE_BUILD>/logs/powerlogs
+/powerlog_YYYY-MM-DD_XXXXXXXX.PLSQL
+
+or from a physical acquisition at:
+/private/var/mobile/Library/BatteryLife/CurrentPowerlog.PLSQL
+Archives/powerlog_YYYY-MM-DD_XXXXXXXX.PLQSQL.gz
+
+Based on the research by Sarah Edwards
+(https://papers.put.as/papers/ios/2016/iOS_of_Sauron_04162016.pdf)
```

Comment:
I believe Joachim mentioned this but creating a ForensicWiki article if possible, is the preferred method to share this information because URLs tend to go stale over time.